examples/stm32/nucleo-h743zi-baremetal/README.md

```diff
@@ -0,0 +1,41 @@
+# Baremetal webserver on NUCLEO-H743ZI
```

Comment:
As soon as we have a tutorial for H7, this README should be a one-line reference to that tutorial. This ensures we have a single place of authority , which does not rot , and things do not go out of sync with each other

Reply:
Good idea, will do the same for F4 and F7